src/ImageGallery.jsx

```diff
@@ -164,11 +164,7 @@ export default class ImageGallery extends React.Component {
 
   componentWillReceiveProps(nextProps) {
     if (this.props.disableArrowKeys !== nextProps.disableArrowKeys) {
-      if (nextProps.disableArrowKeys) {
-        window.removeEventListener('keydown', this._handleKeyDown);
-      } else {
-        window.addEventListener('keydown', this._handleKeyDown);
-      }
+      window[`${nextProps.disableArrowKeys ? 'remove' : 'add'}EventListener`]('keydown', this._handleKeyDown);
```

Comment:
Although this works, this is not preferred and makes code harder to understand.

Reply:
Ok, I thought it would make the code more slender